grammar.js

```diff
@@ -49,19 +51,66 @@ module.exports = grammar(HTML, {
     _node: ($, original) => choice(
       original,
 
+      $._svelte_statement,
+      $.expression,
+      $._svelte_expression,
+    ),
+
+    _svelte_statement: $ => choice(
       $.if_statement,
       $.each_statement,
       $.await_statement,
       $.key_statement,
       $.snippet_statement,
+    ),
 
-      $.expression,
-      $.html_tag,
-      $.const_tag,
-      $.debug_tag,
-      $.render_tag,
+    _single_quoted_attribute_value: $ => repeat1(
+      choice(
+        // Either any escaped character…
+        token(/\\./),
+        // …or any random non-expression piece of string…
+        /[^\\{']+/,
+        // …or an expression.
+        $.expression,
+        $._svelte_expression,
+        $._svelte_statement,
```

Comment:
I don't think `_svelte_expression` of `_svelte_statement` can be in attribute value

Reply:
Ha, you're right. I was trying to make CI happy and I didn't check [the context on this test case](https://github.com/sveltejs/svelte/blob/30b143cef0d5ea17a6edec33496681882eadce34/packages/svelte/tests/validator/samples/logic-block-in-attribute/errors.json). I'll roll this back. Thanks!